util/test/paratest.server

```diff
@@ -748,7 +748,7 @@ sub main {
     }
 
     # Package up the environment
-    my $tmpchplenv = `$ENV{CHPL_HOME}/util/printchplenv --sh`;
+    my $tmpchplenv = `$ENV{CHPL_HOME}/util/printchplenv --simple --all`;
```

Comment:
At first I thought we needed to change tmpchplenv to store them like e.g. export CHPL_HOME='/some/path/' but now I see that stuff is removed a few lines down. Probably worth at least putting a comment indicating that the below removal is no longer relevant - or deleting it entirely.

e.g. replace the for loop with:
```
     for my $line (@lines) {
         my ($key,$value) = split(/=/, $line, 2);
         $chplenv .= "$key=$value ";
}
```